Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Done/DefaultDoneFormButtonStyle.swift

```diff
@@ -1,14 +1,17 @@
 import UIKit
 
 struct DefaultDoneFormButtonStyle: ElementButtonStyle {
-
-    var text: String = "done".localized(forClass: CheckoutTheme.self)
+    var image: UIImage? = nil
```

Comment:
Do we need to assign a nil value its optional if anything not set it will be nil I suppose ?

Reply:
nil is default value and can be overridden. it can be removed but it will not change anything.